### DISCUSSION TITLE
Webpack inline loader config can't work in require-context.macro

### DISCUSSION THREAD

**Discussion by miaolq** [storybookjs/storybook#16480] [Help]
`requireContext('!@svgr/webpack!assets/svgs', false, /\.(svg)$/)` will treat  `!@svgr/webpack!` as path instead of  loader options. How can i solve this problem, thank you.

**Comment by chqcose** [comment]
same problem

```
Module not found: Error: Package path ./?icon&svgo is not exported from package xxx/node_modules/@svgr/webpack
```

**Reply by valentinpalkovic** [reply]
Can you provide more context? Please post your component/story, which uses the inline loader.

**Comment by miaolq** [comment]
https://github.com/storybookjs/require-context.macro/issues/122
